components/index/CardsFeatured/InfectionSummary/Card.vue

```diff
@@ -8,20 +8,26 @@
         title-id="infection-summary"
         :date="date"
       >
+        <template #attentionNote>
+          <i18n
+            path="都外からの持込検体・他県陽性者登録センターによる陽性数」及び新規陽性者の「ワクチン接種状況」 は、全数把握の見直しにより、 2022年9月26日(月曜日)をもって公表を終了しました。なお、検査件数の詳細は{fukushi_web}をご覧ください。"
```

Comment:
```suggestion
            path="都外からの持込検体・他県陽性者登録センターによる陽性数」及び新規陽性者の「ワクチン接種状況」 は、全数届出の見直しにより、 2022年9月26日(月曜日)をもって公表を終了しました。なお、検査件数の詳細は{fukushi_web}をご覧ください。"
```

すみません！指示ミスがありました。「全数把握」ではなく「全数届出」となります 🙇

Reply:
@kaizumaki
レビュー内容コミットしました！